style/git_and_github.qmd

```diff
@@ -1,17 +1,88 @@
-# Using git and GitHub
+# Using Git and GitHub
 
-* Commits should be [atomic](https://dev.to/samuelfaure/how-atomic-git-commits-dramatically-increased-my-productivity-and-will-increase-yours-too-4a84)
-* All code should be committed via pull request to main. **Never** push to main
+-   Commits should be [atomic](https://dev.to/samuelfaure/how-atomic-git-commits-dramatically-increased-my-productivity-and-will-increase-yours-too-4a84)
```

Comment:
Link currently opens in same tab on browser, thoughts on it opening a new tab in browser instead?

Reply:
Interesting point. There are a few ways to do that and may need a change in the yaml: https://github.com/quarto-dev/quarto-cli/discussions/3169